lib/topological_inventory/orchestrator/object_manager.rb

```diff
@@ -115,6 +115,15 @@ def delete_replication_controller(name)
       rescue Kubeclient::ResourceNotFoundError
       end
 
+      def get_collector_image(name)
+        pod = connection.get_pods(
```

Comment:
So this works as is even with the `kube_connection` vs. `ocp_connection` - Or was that issue else where?

Reply:
Just pushed the update for `kube_connection`, I hadn't pushed it yet, sorry!